package.json

```diff
@@ -27,7 +27,7 @@
     "launchdarkly-js-client-sdk": "^2.18.3",
     "lodash": "^4.17.20",
     "remove-accents": "^0.4.2",
-    "sbc-common-components": "2.4.17",
+    "sbc-common-components": "^2.4.18",
```

Comment:
Remove the ^. Its purpose is to auto-upgrade this package every time you npm install. But we don't want it to auto-upgrade without our knowledge (since we need to know/test whatever changed).

Reply:
done.